socorro/processor/breakpad_transform_rules.py

```diff
@@ -261,13 +261,23 @@ def _execute_external_process(self, command_line, processor_meta):
         stackwalker_data.mdsw_status_string = stackwalker_output.get('status', 'unknown error')
         stackwalker_data.success = stackwalker_data.mdsw_status_string == 'OK'
 
+        self.metrics.incr(
+            'run',
+            tags=[
+                'outcome:%s' % ('success' if stackwalker_data.success else 'fail'),
+                'exitcode:%s' % return_code,
+            ]
+        )
+
         if return_code == 124:
-            processor_meta.processor_notes.append('MDSW terminated with SIGKILL due to timeout')
+            msg = 'MDSW terminated with SIGKILL due to timeout'
+            processor_meta.processor_notes.append(msg)
+            self.config.logger.warning(msg)
 
         elif return_code != 0 or not stackwalker_data.success:
-            processor_meta.processor_notes.append(
-                'MDSW failed on "%s": %s' % (command_line, stackwalker_data.mdsw_status_string)
-            )
+            msg = 'MDSW failed with %s: %s' % (return_code, stackwalker_data.mdsw_status_string)
+            processor_meta.processor_notes.append(msg)
+            self.config.logger.warning(msg)
```

Comment:
I changed this. Previously, it would put the command line in the processor notes. The command line is mostly static now since it's defined in code. Also, it seems nuts to me to put the command line in the processor notes which I think are public to everyone. Anyhow, the return code is much more helpful, so I switched this around to add the return code and status string to processor notes and also log them.

Logging is *SUPER* helpful for local development because I can see/grep it without having to fiddle through web pages.